src/components/ProgramViewer.tsx

```
@@ -0,0 +1,79 @@
+                    bookmarked[section.id] 
+                      ? 'bg-blue-100 text-blue-600 scale-105' 
+                      : 'bg-gray-100 text-gray-400 hover:bg-gray-200 hover:text-gray-600'
+                  }`}
+                  aria-label="Bookmark section"
+                >
+                  <Bookmark size={24} fill={bookmarked[section.id] ? "currentColor" : "none"} />
+                </button>
+              </div>
+              
+              {/* Rich Text Output - prose-lg ensures great mobile readability */}
+              <div 
```

ESLint

[error] Delete `·`

(prettier/prettier)